hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -232,6 +234,16 @@ private void validateSecretKeyStatus() throws SCMSecretKeyException {
     }
   }
 
+  @Override
+  public boolean checkAndRotate(boolean force) throws TimeoutException {
+    try {
+      validateSecretKeyStatus();
+    } catch (SCMSecretKeyException e) {
+      e.printStackTrace();
```

Comment:
LOG.error instead

Reply:
This error should be thrown to client-side, like other APIs.